odoo/modules/module.py

```diff
@@ -249,6 +249,14 @@ def module_manifest(path):
         if os.path.isfile(opj(path, manifest_name)):
             return opj(path, manifest_name)
 
+def read_manifest(addons_path, module):
+    mod_path = opj(addons_path, module)
+    manifest_path = module_manifest(mod_path)
+    if manifest_path:
+        with open(manifest_path, 'rb') as fd:
```

Comment:
- don't use bare `open()`, use `with tools.file_open()` for a safer alternative
- in fact, why copy/paste part of the code from `load_information_from_description_file()` instead of *extracting* it, with a proper refactoring? seems you should have kept the use of `tools.file_open()` and the default values for the manifest, while modernizing the code